contributing_to_docs/contributing.adoc

```diff
@@ -14,22 +15,18 @@ There are a few different ways you can contribute to OpenShift documentation:
 
 // * Submit comments at the bottom of each topic (still awaiting implementation)
 * Email the OpenShift documentation team openshift-docs@redhat.com
-* https://github.com/openshift/openshift-docs/issues/new[Create an issue in GitHub]
-* Submit a PR. You can clone the repository, make your changes, and submit a PR.
-This option is best if you have substantial changes. If you open a PR, be sure
-that all of its contents are related and apply to the same versions.
+* Create a link:https://github.com/openshift/openshift-docs/issues/new[GitHub], link:https://issues.redhat.com/secure/CreateIssue!default.jspa[Jira], or link:https://bugzilla.redhat.com/enter_bug.cgi?product=OpenShift%20Container%20Platform[Bugzilla] documentation ticket.
```

Comment:
Are there better URLs to use here?

Reply:
These LGTM.

Reply:
@adellape thank you for your review! I will resolve this comment based on your feedback.